support/docker/production/config/custom-environment-variables.yaml

```diff
@@ -1,3 +1,8 @@
+# 
+# This file will be read by node-config
+# See https://github.com/node-config/node-config/wiki/Environment-Variables#custom-environment-variables
+# 
+
```

Comment:
I discovered that this file was read by `node-config` and found that useful to add as a comment because it was not obvious to me.